homeassistant/config.py

```diff
@@ -554,7 +513,7 @@ def _set_time_zone(hass: HomeAssistant, time_zone_str: Optional[str]) -> None:
             auth_conf,
             mfa_conf))
 
-    await async_load_ha_core_config(hass)
+    await hass.config.load()
```

Comment:
Can we rename this async_load, in line with other function names that are async.

Reply:
Fixed.